### PR TITLE
feat: Add slow fill "grey" list mode to Relayer

### PR DESCRIPTION
**Pull request by nicholaspai**
Enables relayer to mark certain depositors as slow-fill only. Could also use this pattern for a depositor "blacklist"

### CODE REVIEW

File: src/relayer/Relayer.ts [hunk 1]
```diff
@@ -161,6 +161,19 @@ export class Relayer {
         continue;
       }
 
+      // If depositor is on the slow deposit list, then send a zero fill to initiate a slow relay and return early.
+      if (config.slowDepositors.includes(deposit.depositor) && fillCount === 0) {
+        if (sendSlowRelays && tokenClient.hasBalanceForZeroFill(deposit) && fillCount === 0) {
```
Comment (mrice32) [inline src/relayer/Relayer.ts:166]: nit: do we need to re-check fillCount === 0 in this second if?

File: src/relayer/Relayer.ts [hunk 2]
```diff
@@ -161,6 +161,19 @@ export class Relayer {
         continue;
       }
 
+      // If depositor is on the slow deposit list, then send a zero fill to initiate a slow relay and return early.
```
Comment (mrice32) [inline src/relayer/Relayer.ts:164]: nit: should we put this _below_ the limits check (the block below) so this doesn't accidentally push us over the hubpool limits? (I'm not entirely sure whether it's better to just ignore these in that case)

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/relayer/Relayer.ts (hunk 2 above)
Comment (nicholaspai) [reply src/relayer/Relayer.ts:164]: good point. I also want to put it under the empty message check

File: src/relayer/RelayerConfig.ts [hunk 3]
```diff
@@ -53,6 +56,9 @@ export class RelayerConfig extends CommonConfig {
     this.relayerTokens = RELAYER_TOKENS
       ? JSON.parse(RELAYER_TOKENS).map((token) => ethers.utils.getAddress(token))
       : [];
+    this.slowDepositors = SLOW_DEPOSITORS
+      ? JSON.parse(RELAYER_TOKENS).map((token) => ethers.utils.getAddress(token))
```
Comment (mrice32) [inline src/relayer/RelayerConfig.ts:60]:
```suggestion
      ? JSON.parse(SLOW_DEPOSITORS).map((depositor) => ethers.utils.getAddress(depositor))
```

**Review by mrice32 — APPROVED** [review]
LGTM! One copy-paste error

**Review by mrice32 — APPROVED** [review]

**Pull request merged**